src/Numbers.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react"
+function Numbers({state, ans})
+{
+    const [currentState, newState] = state
```

Comment:
I don;t like how you are wrapping everyhting in a jsObject called state because it's misleading

better to explicitly use these props `number, setNumber`